manifests/12_prometheus_rule.yaml

```diff
@@ -65,3 +65,13 @@ spec:
           for: 15m
           labels:
             severity: warning
+    - name: custom-resource-detected
+      rules:
+        - alert: CustomResourceDetected
```

Comment:
Hey, noticed this alert in an openshift cluster. 

The alertname is not very clear, to me custom resource detected can mean any custom resource, having a prefix of MarketplaceCustomResource would be a bit more clear.

Also this is a notification rather than an alert. See https://prometheus.io/docs/practices/alerting/#what-to-alert-on

Also the metric name is wrong, _total indicates a counter rather than a gauge like you have https://prometheus.io/docs/practices/naming/#metric-names

cc @awgreene @anik120 thanks :)